pubtools/_quay/operator_pusher.py

```diff
@@ -523,7 +523,7 @@ def push_index_images(self, iib_results, tag_suffix=None):
         image_schema_tag = "{host}/{namespace}/{repo}:{tag}"
         index_image_repo = image_schema.format(
             host=self.quay_host,
-            namespace=self.target_settings["quay_namespace"],
+            namespace=self.target_settings["quay_operator_repository"].split("/")[-1],
```

Comment:
Namespace of prod is redhat, the one of stage is redhat-pending. I don't think it can be got from quay_operator_repository.